core/runtime/jsruntime.rs

```diff
@@ -232,15 +233,10 @@ pub struct JsRuntime {
   pub(crate) inner: InnerIsolateState,
   pub(crate) allocations: IsolateAllocations,
   extensions: Vec<Extension>,
-  preserve_snapshotted_modules: Option<&'static [&'static str]>,
   event_loop_middlewares: Vec<EventLoopMiddlewareFn>,
-  global_template_middlewares: Vec<GlobalTemplateMiddlewareFn>,
-  global_object_middlewares: Vec<GlobalObjectMiddlewareFn>,
```

Comment:
Why are these fields gone?

Reply:
They were no longer used when I removed multiple realms. They are easy enough to add back if needed in a later pass, but I didn't want to add `allow(unused)`.

Reply:
Surprising - I thought they were used in `ext/node` in Deno repository. Maybe they are just one-off now and don't need to be stored anymore 👍